kernel/rtlil.cc

```diff
@@ -168,25 +170,24 @@ RTLIL::Const RTLIL::Const::from_string(std::string str)
 std::string RTLIL::Const::decode_string() const
 {
 	std::string string;
-	std::vector<char> string_chars;
-	for (int i = 0; i < int (bits.size()); i += 8) {
+	string.reserve(GetSize(bits)/8);
```

Comment:
What about non-multiple-of-8 sizes?

Reply:
The original code does not handle non-multiple-of-8, and it's not the intention of this PR to do so either.

Reply:
OK, gotcha.